### PR TITLE
added level order traversal of binary tree

### PR DESCRIPTION
**Pull request by khushbooagarwal09**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

->>Added C++ code for level order traversal of binary tree. 
->>Constructed binary tree using stack and used queue for printing level order traversal of the binary tree. 
->>algorithm works in 3 steps until queue is empty--
       1. remove the node
       2. save/print the value
       3. add the children to the queue

Fixes#4591

### Checklist

-  [x] I've read the contribution guidelines.  
-  [x] I've checked the issue list before deciding what to submit.
-  [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixed issue num #4591

### CODE REVIEW

File: C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    
+
+
+
+
+
+
+
+
+
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp]: Remove all these extra lines

File: C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+}
+
+// Output of the given binary tree.
+//ith line represents nodes present in ith level
+
+
+// 50
+// 25 75
+// 12 37 62 87
+// 30 70
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp]: Use Multi-Line Comments

File: C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+           }
+           
+          result.push_back(temp);       //adding to the result vector
+        }
+       return result;
+    }
+int main()
+{
+int n=19;
+int arr[]={50,25,12,-1,-1,37,30,-1,-1,-1,75,62,-1,70,-1,-1,87,-1,-1};
```
Comment (HarshCasper) [inline C-Plus-Plus/ds/LevelOrderTraversal_BinaryTree.cpp]: Use dynamic input

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few more changes: 

- Use Comments above the Line of Code not beside it
- Format and Indent the Code properly

**Comment by khushbooagarwal09** [comment]
Ok I will make all the requested changes.
Thank you

**Review by ankitaggarwal23 — COMMENTED** [review]
Issue already available #3408

**Pull request closed without merging**